cmd/kubelet/app/server.go

```diff
@@ -297,8 +297,7 @@ func initConfigz(kc *componentconfig.KubeletConfiguration) (*configz.Config, err
 }
 
 func run(s *options.KubeletServer, kubeDeps *kubelet.KubeletDeps) (err error) {
-	// TODO: this should be replaced by a --standalone flag
-	standaloneMode := (len(s.APIServerList) == 0 && !s.RequireKubeConfig)
+	standaloneMode := s.Standalone
```

Comment:
What happens if someone was relying on enabling "standalone mode" by setting `RequireKubeConfig` and not providing any API servers in the kubeconfig file? Does this change break backwards compatibility?